release notes/v0.27.0.md

```diff
@@ -0,0 +1,137 @@
+- `ramping-arrival-rate`: a variable number of iterations are executed in a specified period of time. This is similar to the ramping VUs executor, but instead of specifying how many VUs should loop through the script at any given point in time, the iterations per second k6 should execute at that point in time can be specified.
+
+It's important to also note that all of these executors, except the `externally-controlled` one, can be used both in local k6 execution with `k6 run`, and in the distributed cloud execution with `k6 cloud`. This even includes "old" executors that were previously unavailable in the cloud, like the `shared-iterations` one. Now, you can execute something like `k6 cloud --iterations 10000 --vus 100 script.js` without any issues.
+
+#### Execution improvements
+
+Multiple execution scenarios can now be configured in a single test run via the new `scenarios` option. These scenarios can run both sequentially and in parallel, and can independently execute different script functions, have different executor types and execution options, and have custom environment variables and metrics tags.
+
+An example using 2 scenarios:
+
+```js
+import http from 'k6/http';
+import { fail } from 'k6';
+
+export let options = {
+  discardResponseBodies: true,
+  scenarios: {
+    my_web_test: {
+      executor: 'constant-vus',
+      exec: 'webtest',
+      vus: 50,
+      duration: '30s',
+      tags: { my_test_version: 'v2' },
+      env: { TESTVERSION: 'v2' },
+    },
+    my_api_test: {
+      executor: 'ramping-arrival-rate',
+      exec: 'apitest',
+      startTime: '30s',
+      startRate: 50,
+      timeUnit: '1s',
+      preAllocatedVUs: 50,
+      maxVUs: 100,
+      stages: [
+        { target: 200, duration: '30s' },
+        { target: 0, duration: '30s' },
+      ],
+      tags: { my_test_version: 'v1' },
+      env: { TESTVERSION: 'v1' },
+    },
+  }
+};
+
+export function webtest() {
+  if (__ENV.TESTVERSION != 'v2') fail();
+  http.get('https://test.k6.io/contacts.php');
+}
+
+export function apitest() {
+  if (__ENV.TESTVERSION != 'v1') fail();
+  http.get('https://test-api.k6.io/');
+}
+```
+
+As shown in the example above and the [documentation](https://k6.io/docs/using-k6/scenarios), all executors have some additional options that improve their flexibility and facilitate code reuse, especially in multi-scenario test runs:
+- Each executor has a `startTime` property, which defines at what time relative to the test run start it will start executing. In the above example, the `my_api_test` scenario will start after `my_web_test` completes.
```

Comment:
I think that code comments will be cleaner and easier to follow, compared to references from this list to the script above

```suggestion
- Each executor has a `startTime` property, which defines at what time, relative to the beginning of the whole test run, the scenario will start being executed. 
```